doc/Analytics.xml

```diff
@@ -3397,127 +3397,71 @@ Added Recognition rule engines</revremark>
     </section>
     <section>
       <title>Counting rule</title>
-      <para>Counting rule counts the number of motion object passing through the set of line segments (barrier) and optional direction attribute. The configuration parameters also include the time interval to report the events and time interval to reset its counter. </para>
+      <para>Counting rule counts the number of objects passing through a set of line segments
+        (barriers) with an optional direction. The configuration parameters also include the time
+        interval to report the events and time interval to reset its counter.<variablelist>
+          <varlistentry>
+            <term>Parameters</term>
+            <listitem>
+              <para role="param">Segments [tt:Polyline]</para>
+              <para role="text">One or more polylines used for detection. An object must have passed
+                through all of them to register as a count. ??? or ??? Passing through any of the
```

Comment:
I have been working through the text and would approve the changes but can you explain the usage of the ???.